file_formats/ie_formats/ini.htm

```diff
@@ -10,405 +10,20 @@
-        <br />
-
-        <strong>exit = name_of_group_section</strong><br />
-        This attribute expects one or more comma-separated names of group sections. It is executed by the engine whenever the player leaves the area. It is only supported by PST, IWD and IWD2.<br />
-        <br />
-
-        <strong>event = name_of_group_section</strong><br />
-        This attribute expects one or more comma-separated names of group sections. It is executed by the engine whenever the conditions defined by the assigned sections are met.<br />
-        <br />
-      </div>
+      <a href="ini_spawn.htm">Creature spawning</a> (PST, IWD, IWD:HoW, IWD:TotL, IWD2, BGEE)</span><br />
       <br />
-
-
-      <strong>[name_of_group_section]</strong><br />
-      <div class="indent1">
-        This section controls a group of individual spawn sections. It supports the following attributes:<br />
-        <br />
-
-        <strong>critters = name_of_creature_section1,name_of_creature_section2,...</strong><br />
-        This attribute expects a comma-separated sequence of creature section names. These sections define individual creatures with their spawn conditions, attributes and behavior. Each listed creature section is evaluated and executed when their spawn conditions are met.<br />
-        <br />
-
-        <strong>interval = number</strong><br />
-        Time interval, measured in AI ticks (15 ticks = 1 second), before creature sections defined by the "critters" attribute are evaluated. This attribute is only relevant if the section is linked to the "event" attribute of the [spawn_main] section. The smallest supported interval value appears to be 15 AI ticks. Smaller values are adjusted accordingly by the engine.<br />
-        <br />
-
-        <strong>detail_level = name</strong><br />
-        This was possibly a memory saving option, but eventually got excluded — it is controlled by the "Detail level" key not present by default in torment.ini. Known attribute values: <span class="code">High, Medium, Low</span>. However the engine has a bug and treats Medium entries as High. It is only supported by PST.<br />
-        <br />
-
-        <strong>control_var = scope::variable_name</strong><br />
-        This attribute was supposed to control whether enter and exit spawn groups are executed by checking a boolean value defined by the specified variable. <strong>Note:</strong> Specifying this attribute causes the game to crash because of a buggy implementation.<br />
-        <br />
-
-        <strong>spawn_time_of_day = string</strong><br />
-        This attribute was supposed to be a bit-string with allowed times for enter and exit spawn groups to execute. It appears to be unused by the engine.<br />
-        <br />
-      </div>
+      <a href="ini_anim.htm">Creature animations</a> (BGEE)</span><br />
```

Comment:
```suggestion
      <a href="ini_anim.htm">Creature animations</a> (PST, BGEE)</span><br />
```

Reply:
In that case I should add a short paragraph about the "original" PST-style animation definition, too.